bin/tbg

```diff
@@ -497,8 +497,13 @@ cp -a "$TBG_cfgFile" tbg/submit.cfg
 #warn, if there are still unresolved !TBG_ variables left
 check_final tbg/submit.start "$tooltpl_file_data"
 
-if [ -n "$submit_command" ] ; then
-    $submit_command tbg/submit.start
-else
-    echo "nothing to submit (-s option set?)"
+# verify picongpu call
+bash tbg/submit.start --validate
```

Comment:
According to @psychocoderHPC this is not allowed since `tbg` should be independent of PIConGPU.

Reply:
As long as this is done conditionally, any code could support `--validate`.

I would suggest to add a tbg optional CLI argument for this.

Reply:
I think the problem is that we can't really check if the program that is called by `tbg` supports `--validate`, if not we would just run the program without an allocation. In that case, it would be the users' responsibility to disable the flag. I think the idea to put it in `submitAction.sh` is really good, that makes it independent of `tbg` and we can maybe control it through a PIConGPU environment variable.